### ISSUE TITLE
ReLU applied to logits

### ISSUE THREAD

**Issue by OccamsRazorResearch** [huyng/tensorflow-vgg#13]
you're applying a relu to your logits

https://github.com/huyng/tensorflow-vgg/blob/master/vgg.py#L59
